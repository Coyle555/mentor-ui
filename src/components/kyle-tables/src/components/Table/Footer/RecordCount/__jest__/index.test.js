import React from 'react';
import { RecordCount } from '../index';
import renderer from 'react-test-renderer';

test('Default render of record count', () => {
	const tree = renderer.create(<RecordCount />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Record count render with no viewable entries', () => {
	const tree = renderer.create(<RecordCount entriesViewable={0} />).toJSON();

	expect(tree).toMatchSnapshot();
});

