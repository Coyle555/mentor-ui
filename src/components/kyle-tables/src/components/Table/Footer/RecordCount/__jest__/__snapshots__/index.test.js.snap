// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default render of record count 1`] = `
<p>
  Showing 
  <b>
    1
  </b>
   to 
  <b>
    25
  </b>
   of 
  <b>
    0
  </b>
   records
</p>
`;

exports[`Record count render with no viewable entries 1`] = `
<p>
  Showing 
  <b>
    0
  </b>
   to 
  <b>
    0
  </b>
   of 
  <b>
    0
  </b>
   records
</p>
`;
